src/summary/cards/summary-card.component.tsx

```diff
@@ -15,14 +19,14 @@ export default function SummaryCard(props: SummaryCardProps) {
     </div>
   );
 }
-
-SummaryCard.defaultProps = {
-  styles: {}
-};
+SummaryCard.defaultProps = { cardSize: "21.75rem" };
 
 type SummaryCardProps = {
   name: string;
   match: match;
   children: React.ReactNode;
+  cardSize: String;
```

Comment:
this shouldn't be necessary, you can pass in the `width` as part of the `styles` prop